packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -618,6 +618,8 @@ export async function createHotReloaderTurbopack(
 
         const errors: CompilationError[] = []
 
+        currentEntryIssues.delete(getEntryKey('app', 'server', '_error'))
+        currentEntryIssues.delete(getEntryKey('pages', 'server', '_error'))
```

Comment:
This feels wrong to me, I also don't think `app` has a `_error` route.

We might just want to subscribe to changes of the `_error` page.